core/metamodel/src/main/java/org/apache/isis/core/metamodel/interactions/managed/ManagedAction.java

```diff
@@ -158,7 +160,12 @@ public ManagedObject invokeWithRuleChecking(
     // -- INVOKE HELPER
 
     private boolean isValueTypeMixin() {
-        return getOwner().getSpecification().isValue();
+        return getOwner().getSpecification().isValue()
+                && !getOwner().getSpecification().lookupFacet(ValueFacet.class)
```

Comment:
maybe introduce a new bean-kind  to check that very early - or otherwise refactor to be cleanup semantics